risc0/zkvm/src/host/server/exec/syscall.rs

```diff
@@ -624,4 +610,24 @@ impl<'a> PosixIo<'a> {
         writer.borrow_mut().write_all(from_guest_bytes.as_slice())?;
         Ok((0, 0))
     }
+
+    fn sys_log(&mut self, ctx: &mut dyn SyscallContext) -> Result<(u32, u32)> {
+        let buf_ptr = ctx.load_register(REG_A3);
+        let buf_len = ctx.load_register(REG_A4);
+        let from_guest = ctx.load_region(buf_ptr, buf_len)?;
+        // write to stdout, but be sure to point it to where the file descriptor is pointing
+        let fd = ctx.load_register(REG_A3);
```

Comment:
I don't understand this line or the comment. It looks like we're not using `fd` for anything